### PR TITLE
Relative references, removed tweet link

### PR DESCRIPTION
**Pull request by marians**
RFR @oponder @puja108 

- Made the references work even when the page is displayed under a path different than `/ `
- Changed the link to giantswarm.io

### CODE REVIEW

File: content/index.html [hunk 1]
```diff
@@ -51,15 +51,15 @@
...
+      <p><a href="https://giantswarm.io/"><img src="giantswarm_logo_standard_white.svg" width="100" height="50" /></a></p>
 
       <h1>Hello World</h1>
 
       <p>Congratulations! You have just deployed and started a little something on Giant Swarm.</p>
 
       <p>Now go ahead and learn how easy it is to create and run your own service <a href="https://docs.giantswarm.io/guides/your-first-service/">in your favorite language</a>.</p>
 
-      <p>Or share your pride and let the world know by <a target="_blank" href="https://twitter.com/share?text=I've just deployed HelloWorld on @giantswarm!"/>tweeting this</a>!</p>
+      <p>Or share your pride and let the world know by <a target="_blank" href="https://twitter.com/share?url=https://giantswarm.io/&amp;text=I've just deployed my first service on @giantswarm!"/>tweeting this</a>!</p>
```
Comment (puja108) [inline content/index.html]: maybe instead of `on @giantswarm` we could do `on my own @giantswarm #Kubernetes cluster` or sth like that

Comment (marians) [inline content/index.html]: Hmm. Since we still use the image for legacy stuff, I'd rather not mention Kubernetes now.

Comment (puja108) [inline content/index.html]: I see, true, we can change it later once we do the move.

**Comment by puja108** [comment]
LGTM

**Pull request merged**